File: services/sending_mail/sending_mail.py
import requests
from pprint import pprint

headers = {'Mailsac-Key': 'YOUR_API_KEY_HERE'}
url = 'https://mailsac.com/api/outgoing-messages'
mail = { 'to':'myfriend@mailsac.com',
         'from':'user1@mailsac.com',
         'subject':'Hello Myfriend',
         'text': 'mailsac allows for sending of email'
       }

r = requests.post(url, data=mail, headers=headers)
pprint(r.json())
'''
{'from': 'user1@mailsac.com',
 'id': 'fe-tgitrk8dbga',
 'to': ['myfriend@mailsac.com']}
'''

File: services/sending_mail/sending_mail.js
const superagent = require('superagent')

superagent
  .post('https://mailsac.com/api/outgoing-messages')
  .set('Mailsac-Key', 'YOUR_API_KEY_HERE')
  .send({
      to: "myfriend@mailsac.com",
      from: "user1@mailsac.com",
      subject: "Hello Myfriend",
      text: "test message from mailsac"
  })
  .then((res) => {
      console.log(res.body)
  })
  .catch(err => {
      console.log(err.message)
  })

  /*
  {
  from: 'user1@mailsac.com',
  to: [ 'myfriend@mailsac.com' ],
  id: 'fe-ipic46bqge8'
}
*/
